### ISSUE TITLE
Erroneous “Incorrect indentation; expected 3 space, found 4”

### ISSUE THREAD

**Issue by westonruter** [WordPress-Coding-Standards/WordPress-Coding-Standards#133]
The conditional [here](https://github.com/x-team/wp-widget-customizer/blob/5efcda2c687327fd75efaec2685e09acebf32025/widget-customizer.php#L417) is causing an error unexpectedly:

```php
$icon_path = sprintf( plugin_dir_path( __FILE__ ) . 'icons/icon-%s.png', $available_widget['id_base'] );
$icon_url  = plugin_dir_url( __FILE__ ) . 'icons/icon-default.png';
if ( file_exists( $icon_path ) ) {
    $icon_url = sprintf( plugin_dir_url( __FILE__ ) . 'icons/icon-%s.png', $available_widget['id_base'] );
}
```

Here's the error:

```
FILE: ...-develop/src/wp-content/plugins/widget-customizer/widget-customizer.php
--------------------------------------------------------------------------------
FOUND 1 ERROR(S) AFFECTING 1 LINE(S)
--------------------------------------------------------------------------------
 417 | ERROR | Incorrect indentation; expected 3 space, found 4
     |       | (WordPress.WhiteSpace.ScopeIndent.Incorrect)
--------------------------------------------------------------------------------
```

The issue happens in both `master` and `develop`.

**Comment by JDGrimes** [comment]
I can't reproduce this anymore, and the upstream version of the `ScopeIndent` sniff is being used now instead (see #250). Closing.
